.ruby-version

```diff
@@ -0,0 +1 @@
+2.7.2
```

Comment:
Why is this `.ruby-version` file here? I don't think it is supposed to be

Reply:
Accidental check in.  Good catch!